browser/src/control/Control.UIManager.ts

```diff
@@ -471,6 +471,24 @@ class UIManager extends window.L.Control {
 		app.serverConnectionService.onBasicUI();
 	}
 
+	initializeBackstageView(): void {
```

Comment:
Starter screen does not handle dark mode right now.
maybe `this.initDarkModeFromSettings();` is needed somewhere here?